src/main/scala/example/TestHarness.scala

```diff
@@ -36,6 +40,82 @@ class TestHarness(implicit val p: Parameters) extends Module {
 }
 
 object Generator extends GeneratorApp {
+  //Copied from rocketchip
```

Comment:
Do we not inherit this from rocketchip by using the `GeneratorApp` trait?

Reply:
I tried doing this some fancy way but since this branch is not long for the world I'm not gonna worry about it.

Reply:
fair enough